dashboard/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Use the official Python base image
+FROM python:latest
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy the requirements file into the container at /app
+COPY requirements.txt /app/
```

Comment:
Side note:

My .sh scripts that run my Dockerfiles have to transfer requirements.txt from root into the working dir, and then after delete it. 

It works but it's very kak-handed and bloats the container with needed modules.